### PR TITLE
Added Vagrant support

### PR DESCRIPTION
**Pull request by Manouchehri**
Requires VMware Fusion 10 or Workstation 14.

### CODE REVIEW

**Comment by rocallahan** [comment]
I don't know anything about Vagrant but I guess this is fine.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		cd rr
+		mkdir obj
+		cd obj
+		cmake ..
+		make -j8
+		make test
+	SHELL
+
+	config.vm.provision "shell", inline: <<-SHELL
+		cd /home/vagrant/rr/ojb/
```
Comment (rocallahan) [inline Vagrantfile]: Actually, `obj` doesn't look right

Comment (Manouchehri) [reply Vagrantfile]: Oops, fat fingered that right before committing.

**Comment by Manouchehri** [comment]
@rocallahan It's mostly for Mac and Windows VMware users who use [Vagrant](https://www.vagrantup.com/) to automate their dev setups.

**Pull request merged**